test/onnx/dynamo/test_exporter_api.py

```diff
@@ -7,13 +7,13 @@
 import torch
 from beartype import roar
 from torch.onnx import dynamo_export, ExportOptions, ExportOutput
-from torch.onnx._internal import exporter
 from torch.onnx._internal.exporter import (
     _DEFAULT_OPSET_VERSION,
+    _ResolvedExportOptions,
```

Comment:
```suggestion
    ResolvedExportOptions,
```

If a name is to be imported by other modules, we should mark as public. (Itself is still considered private to onnx because its in `_internal`)

Reply:
Basically private members (of a module) shouldn't appear on import lines.

Reply:
second this.